csr/csr.go

```diff
@@ -49,7 +49,7 @@ type BasicKeyRequest struct {
 
 // NewBasicKeyRequest returns a default BasicKeyRequest.
 func NewBasicKeyRequest() *BasicKeyRequest {
-	return &BasicKeyRequest{"ecdsa", curveP256}
+	return &BasicKeyRequest{"rsa", 2048}
```

Comment:
I'd still prefer to keep the ECDSA default; the RSA is provided in print-defaults to make it clear how to override the algorithm.